plugins_src/import_export/wpc_yafaray.erl

```diff
@@ -4691,7 +4691,7 @@ export_faces(F, [#e3d_face{mat=[Mat|_],tx=Tx,vs=[A,B,C],vc=VCols}|T],
                              [length(VCols)]),
                    ""
            end,
-    uniprintln(F, "~s    <f a=\"~s\" b=\"~s\" c=\"~s\"~s~s",
+    uniprintln(F, "~ts    <f a=\"~ts\" b=\"~ts\" c=\"~ts\"~ts~ts",
```

Comment:
These should not contain unicode characters, or?

Reply:
In fact, in a deep look, only the first one can contain unicode since it stores the material name (_Shader_). The others are numbers already formatted and converted to a list.